tests/meta/test_ensure_type_hints.py

```diff
@@ -49,9 +49,10 @@ def pytest_generate_tests(metafunc: _pytest.python.Metafunc) -> None:
             if module_name == "gitlab.base":
                 continue
 
-            class_info_list.append((class_name, class_value))
-
-    metafunc.parametrize("class_info", class_info_list)
+            if not class_name.endswith("Manager"):
+                continue
+            class_info_set.add((class_name, class_value))
+    metafunc.parametrize("class_info", class_info_set)
```

Comment:
Think we lost a few newlines here from before :D
```suggestion
            if not class_name.endswith("Manager"):
                continue

            class_info_set.add((class_name, class_value))

    metafunc.parametrize("class_info", class_info_set)
```

Reply:
Done